source/jormungandr/jormungandr/interfaces/v1/serializer/status.py

```diff
@@ -131,7 +131,52 @@ def get_backend_class(self, obj):
 
 
 class RidesharingServicesSerializer(OutsideServiceCommon):
-    pass
+    rating_scale_min = MethodField(schema_type=int, display_none=False)
+    rating_scale_max = MethodField(schema_type=int, display_none=False)
+    crowfly_radius = MethodField(schema_type=int, display_none=False)
+    network = MethodField(schema_type=str, display_none=False)
```

Comment:
minor : Why can't those just be `Field` ? 
As `Field` you don't need to write specific functions  for each of them.